docs/tutorials/geomesa-quickstart-lambda.rst

```diff
@@ -0,0 +1,141 @@
+GeoMesa Lambda Quick Start
```

Comment:
added it to the tutorials - didn't want to start over on this file with the sync, so we can iron out any discrepancies with the sync next time